config/install/embed.button.media.yml

```diff
@@ -0,0 +1,19 @@
+langcode: en
+status: true
+dependencies:
+  content:
+    - 'file:file:39004672-000c-4b13-af59-838822a4bc5a'
+  module:
+    - entity_embed
+    - media_entity
+label: Media
+id: media
+type_id: entity
+type_settings:
+  entity_type: media
+  bundles: {  }
+  display_plugins: {  }
```

Comment:
I'd limit available plugins here. Only thing that makes sense in my opinion is label and the one that renders entity in dedicated "Embed" view mode. 

We should create that view mode and configure it to have decent look and feel with all bundles.